src/package.json

```diff
@@ -6,8 +6,8 @@
   "typings": "cast.d.ts",
   "nativescript": {
     "platforms": {
-      "android": "5.4.0",
-      "ios": "5.4.2"
+      "android": "5.3.1",
+      "ios": "5.3.1"
```

Comment:
Why were these not updated to 6.0.1?

Reply:
good find. Will update it. 
Ran into a crash (testing on iOS 13) which may be related to latest xcode, so trying to figure that out as well - probably we have to upgrade the GoogleCast.framework (it's recommended for iOS 13).

Reply:
updated.
Also upgrading to GoogleCast 4.4.5. Dynamic cast API for iOS fixed the crash on iOS 13 device.